common/djangoapps/student/models.py

```diff
@@ -87,6 +87,13 @@ def __unicode__(self):
         return self.name
 
 
+class School(models.Model):
+    name = models.CharField(max_length=256)
```

Comment:
max_length=256 is a normal value, but developers set 254 for mysql `varchars` fields 255 - 1(extra byte)

Reply:
please add Meta sort field

Reply:
please set `name` as unique field